### PR TITLE
Staging 1.5 client

### PR DESCRIPTION
**Pull request by caesarxuchao**
Created the 1.5 folder and remove the 1.4 folder in the staging area in the master branch.

Content of kubernetes/client-go/1.4 will be pulled from the kubernetes/kubernetes 1.4 branch (https://github.com/kubernetes/contrib/pull/1719)

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/32906)

### CODE REVIEW

**Comment by caesarxuchao** [comment]
@k8s-bot test this, issue #IGNORE

**Comment by lavalamp** [comment]
Can you make the commit in the middle do a rename instead of having a delete & then recreate commit?

Also test is red?

Then lgtm.

**Comment by caesarxuchao** [comment]
I merged the commit, but github doesn't show them as renaming, maybe it's because too many files are moved.

**Comment by lavalamp** [comment]
I think you may have dropped the second commit instead of squashing them, github says only +7 lines right now...

**Comment by caesarxuchao** [comment]
No, I didn't drop it. Github is broken.

File: test/e2e/disruption.go [hunk 1]
```diff
@@ -22,13 +22,13 @@ import (
...
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	release_1_4 "k8s.io/client-go/1.4/kubernetes"
-	apiapi "k8s.io/client-go/1.4/pkg/api"
-	"k8s.io/client-go/1.4/pkg/api/unversioned"
-	api "k8s.io/client-go/1.4/pkg/api/v1"
-	extensions "k8s.io/client-go/1.4/pkg/apis/extensions/v1beta1"
-	policy "k8s.io/client-go/1.4/pkg/apis/policy/v1alpha1"
-	"k8s.io/client-go/1.4/pkg/util/intstr"
+	release_1_4 "k8s.io/client-go/1.5/kubernetes"
```
Comment (mml) [inline test/e2e/disruption.go]: Bad name for this import.

**Review by mml — CHANGES_REQUESTED** [review]

**Comment by mml** [comment]
@caesarxuchao Also I think you want release-note-none.

File: test/e2e/disruption.go (hunk 1 above)
Comment (caesarxuchao) [inline test/e2e/disruption.go]: Nice catch. I remove the rename.

**Comment by caesarxuchao** [comment]
Can I get the label? Thanks.

**Review by mml — APPROVED** [review]

**Comment by mml** [comment]
@lavalamp git doesn't distinguish actually have a way to track file rename/move operations.  Instead it does "rename detection" when it tries to summarize a change to you.

LGTM

**Comment by caesarxuchao** [comment]
@k8s-bot e2e test this, issue #33175.

**Pull request merged**